src/renderer/components/LoginComponent/LoginComponent.tsx

```diff
@@ -1,10 +1,17 @@
 import React from 'react';
+import { FaGithub } from 'react-icons/fa';
 import './LoginComponent.scss';
 
 const LoginComponent = () => {
 
   return (
-    <div>LoginComponent</div>
+    <div className="container">
+    <div>Welcome to VNlab</div>
+    <div>
+    <button className="button"onClick={()=>{console.log('Prompt new browser window with GitHub auth.')}}>Login to GitHub <FaGithub />
+</button>
+    </div>
+    </div>
   );
```

Comment:
Please format this file as well, DOM is much easier to read if the elements are actually visibly nested.  
You can try out Prettier if you'd like.